llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -118,6 +118,12 @@ Register RISCVInstrInfo::isLoadFromStackSlot(const MachineInstr &MI,
   case RISCV::FLD:
     MemBytes = 8;
     break;
+  case RISCV::VL8RE8_V:
```

Comment:
What about 1/2/4? And also Spill/Reload pseudos?

Reply:
Not all the sizes showed up in tests with asserts. I didn't see any of the spill/reload pseudos appear either

Reply:
Oh, you just added them! We still need handle `PseudoVSPILL`/`PseudoVRELOAD` I think.

Reply:
I never see the spill pseudos appear here for some reason

Reply:
We may don't have ehough test coverages for segment types.